Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -3872,11 +3872,17 @@ void CClientGame::PostWorldProcessHandler()
     m_pRootEntity->CallEvent("onClientPreRender", Arguments, false);
 }
 
-void CClientGame::PostWorldProcessPedsAfterPreRenderHandler()
+void CClientGame::PostWorldProcessEntitiesAfterPreRenderHandler()
 {
     CLuaArguments Arguments;
-    m_pRootEntity->CallEvent("onClientPedsProcessed", Arguments, false);
 
+    if (m_pRootEntity->CallEvent("onClientPedsProcessed", Arguments, false))
+    {
+        CStaticFunctionDefinitions::AddEventHandler("onClientPedsProcessed", "onClientPostUpdate", "The 'onClientPedsProcessed' event is deprecated and has been renamed to 'onClientPostUpdate'.");
+        return; 
+    }
```

Comment:
I don't know how you came up with the idea to put that here, remove it and adapt `CStaticFunctionDefinitions::AddEventHandler`.